src/ncp/ncp_base_mtd.cpp

```diff
@@ -2551,6 +2553,51 @@ template <> otError NcpBase::HandlePropertyGet<SPINEL_PROP_CNTR_ALL_IP_COUNTERS>
     return error;
 }
 
+#if OPENTHREAD_CONFIG_MAC_RETRY_SUCCESS_HISTOGRAM_ENABLE
+template <> otError NcpBase::HandlePropertyGet<SPINEL_PROP_CNTR_MAC_RETRY_HISTOGRAM>(void)
+{
+    otError         error = OT_ERROR_NONE;
+    const uint32_t *histogramDirect;
+    const uint32_t *histogramIndirect;
+    uint8_t         histogramDirectEntries;
+    uint8_t         histogramIndirectEntries;
+
+    histogramDirect   = otLinkGetTxDirectRetrySuccessHistogramGet(mInstance, &histogramDirectEntries);
+    histogramIndirect = otLinkGetTxIndirectRetrySuccessHistogramGet(mInstance, &histogramIndirectEntries);
+
+    if (histogramDirectEntries)
```

Comment:
Please use parenthesis for single if-statement:

```
if (histogramDirectEntries)
{
..
}
```

Reply:
Also instead of if-statement, it's easier just to call:

assert(histogramDirect);